cmd/main.go

```diff
@@ -52,7 +52,7 @@ import (
 )
 
 func init() {
-	strfmt.MarshalFormat = strfmt.ISO8601LocalTime
+	strfmt.MarshalFormat = strfmt.RFC3339Millis
```

Comment:
Does posgres support this format?

Reply:
Here's an example of what's being returned by gorm `CreatedAt:2020-09-13T06:26:46.000Z`, before being marshalled by Swagger. The JSON looks like `{ "created_at": "2020-09-13T06:26:46.000Z" }`. In any case, I've tested this change E2E including the UI.

Reply:
So what is the difference?

Reply:
Compare with `strfmt.ISO8601LocalTime`, where Swagger drops the UTC designation making it a local time `{"created_at": "2020-09-13T06:26:46"}`.

Reply:
> So what is the difference?

Local time vs explicitly specifying UTC offset https://en.wikipedia.org/wiki/ISO_8601

Reply:
So the change is how the time returned from swagger?

Reply:
Correct. See https://github.com/mareklibra/facet-lib/pull/198 and MGMT-1015.